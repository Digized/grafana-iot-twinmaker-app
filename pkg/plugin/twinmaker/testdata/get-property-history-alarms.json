{
    "NextToken": null,
    "PropertyValues": [
        {
            "EntityPropertyReference": {
                "ComponentName": "AlarmComponent",
                "EntityId": "Mixer_1_4b57cbee-c391-4de6-b882-622c633a697e",
                "ExternalIdProperty": {
                    "alarm_key": "Mixer_1_597c735b-38fd-476c-b276-7592b1699ef8"
                },
                "PropertyName": "alarm_status"
            },
            "Values": [
                {
                    "Time": "2022-04-27T17:19:42Z",
                    "Timestamp": null,
                    "Value": {
                        "BooleanValue": null,
                        "DoubleValue": null,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": "NORMAL"
                    }
                }
            ]
        }
    ]
}