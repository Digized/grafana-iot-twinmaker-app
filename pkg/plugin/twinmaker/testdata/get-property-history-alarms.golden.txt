ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "custom": {}
}
Name: 
Dimensions: 2 Fields by 1 Rows
+---------------------------------------------------------------------------------------------+-------------------------------+
| Name: alarm_status                                                                          | Name: Time                    |
| Labels: componentName=AlarmComponent, entityId=Mixer_1_4b57cbee-c391-4de6-b882-622c633a697e | Labels:                       |
| Type: []*string                                                                             | Type: []*time.Time            |
+---------------------------------------------------------------------------------------------+-------------------------------+
| NORMAL                                                                                      | 2022-04-27 17:19:42 +0000 UTC |
+---------------------------------------------------------------------------------------------+-------------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////KAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAxP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADk/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAAT///8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAAAgAAAHwAAAAEAAAAnv///xQAAAA8AAAARAAAAAAACgFEAAAAAQAAAAQAAABc////CAAAABAAAAAEAAAAVGltZQAAAAAEAAAAbmFtZQAAAAAAAAAAAAAGAAgABgAGAAAAAAADAAQAAABUaW1lAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAMwAAADQAAAAAAAFAcwAAAACAAAAPAAAAAQAAADU////CAAAABgAAAAMAAAAYWxhcm1fc3RhdHVzAAAAAAQAAABuYW1lAAAAAAgADAAIAAQACAAAAAgAAABoAAAAXAAAAHsiY29tcG9uZW50TmFtZSI6IkFsYXJtQ29tcG9uZW50IiwiZW50aXR5SWQiOiJNaXhlcl8xXzRiNTdjYmVlLWMzOTEtNGRlNi1iODgyLTYyMmM2MzNhNjk3ZSJ9AAAAAAYAAABsYWJlbHMAAAAAAAAEAAQABAAAAAwAAABhbGFybV9zdGF0dXMAAAAAAAAAAP/////IAAAAFAAAAAAAAAAMABYAFAATAAwABAAMAAAAGAAAAAAAAAAUAAAAAAAAAwQACgAYAAwACAAEAAoAAAAUAAAAaAAAAAEAAAAAAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAgAAAAAAAAABgAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAIAAAAAAAAAAAAAAACAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAABgAAAE5PUk1BTAAAAEze7n7Q6RYQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAOAAAAAAABAABAAAAOAIAAAAAAADQAAAAAAAAABgAAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAIAAAAADAAAATAAAACgAAAAEAAAAxP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAADk/v//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAAT///8IAAAAGAAAAA0AAAB7ImN1c3RvbSI6e319AAAABAAAAG1ldGEAAAAAAgAAAHwAAAAEAAAAnv///xQAAAA8AAAARAAAAAAACgFEAAAAAQAAAAQAAABc////CAAAABAAAAAEAAAAVGltZQAAAAAEAAAAbmFtZQAAAAAAAAAAAAAGAAgABgAGAAAAAAADAAQAAABUaW1lAAASABgAFAATABIADAAAAAgABAASAAAAFAAAAMwAAADQAAAAAAAFAcwAAAACAAAAPAAAAAQAAADU////CAAAABgAAAAMAAAAYWxhcm1fc3RhdHVzAAAAAAQAAABuYW1lAAAAAAgADAAIAAQACAAAAAgAAABoAAAAXAAAAHsiY29tcG9uZW50TmFtZSI6IkFsYXJtQ29tcG9uZW50IiwiZW50aXR5SWQiOiJNaXhlcl8xXzRiNTdjYmVlLWMzOTEtNGRlNi1iODgyLTYyMmM2MzNhNjk3ZSJ9AAAAAAYAAABsYWJlbHMAAAAAAAAEAAQABAAAAAwAAABhbGFybV9zdGF0dXMAAAAAUAIAAEFSUk9XMQ==
